torchreid/integration/sc/task.py

```diff
@@ -268,6 +268,7 @@ def train(self, dataset: Dataset, output_model: ModelEntity, train_parameters: O
         else:
             train_model = deepcopy(self._model)
 
+        self._cfg.train.lr = self._hyperparams.learning_parameters.learning_rate
```

Comment:
All the hyperparameters should be declared and documented in `torchreid/integration/sc/parameters.py` and `torchreid/integration/sc/configuration.yaml `. At the moment I don't see `learning_rate` among the declared papameters.

Reply:
I added `learning_rate` in `torchreid/integration/sc/parameters.py` as you reviewed. But there is already `learning_rate` in `torchreid/integration/sc/configuration.yaml@L45`. Thank you!